### DISCUSSION TITLE
Enhancement: Wiki on the Github

### DISCUSSION THREAD

**Discussion by aflacc** [FunkinCrew/Funkin#1625] [Ideas]
I feel like there should be a wiki on the Github that could further explain whats going on or how to do certain things. Such as maybe how to add weeks or how to use cutscenes. Heres a few examples of wiki pages that could be on here:
- how to use the Debug Menu
- how to add a week or song
- how to change the health loss
- how to change a characters position
- how to change the camera position

**Comment by oketariq** [comment]
Gamebanana has a lot of really cool tutorials explaining most of those things.
https://gamebanana.com/tuts/games/8694

**Comment by TentaRJ** [comment]
I've been making tutorials on GameBanana actually. I'm working on a full guide, but I'm planning on releasing it on the next major update.

**Comment by isakube** [comment]
> I've been making tutorials on GameBanana actually.


Animation Debug Menu when? It's probably not very useful if you don't have access to source but (bro github wrecks my shrugboi)
